modules/tiles/src/tileset/tile-3d.ts

```diff
@@ -341,6 +342,24 @@ export class Tile3D {
     this._selectedFrame = 0;
   }
 
+  /**
+   * Memory usage of tile on GPU
+   */
+  _getGpuMemoryUsageInBytes(): number {
+    switch (this.type) {
+      case TILE_TYPE.MESH:
+      case TILE_TYPE.POINTCLOUD:
+        return this.content.byteLength;
```

Comment:
Probably, it would be nice to set `content.gpuMemoryUsageInBytes = content.byteLength` for `POINTCLOUD` in `3d-tiles`.